projects/tuist/features/step_definitions/shared/tuist.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 require "open3"
 Given(/tuist is available/) do
-  system("swift", "build")
-  @tuist = ".build/debug/tuist"
-  @tuistenv = ".build/debug/tuistenv"
+  if ENV["CI"].nil?
+    system("swift", "build", "-c", "release", "--product", "tuist", "--product", "tuistenv", "--product", "ProjectDescription", "--product", "ProjectAutomation")
+  end
+  FileUtils.cp_r("projects/tuist/vendor", ".build/release/vendor")
```

Comment:
This part I am not a big fan of.
We needed it because release build needs to find vendor binaries & templates, with debug build that is done by a special logic in the `#if DEBUG`.
If any of you have any suggestion on how to make it better, it's more then welcomed 😄

Reply:
I thought about using a custom compiler directive but I think that'd be more hassle than worth (I am also not 100 % sure it'd be possible).

I believe the current solution is fine but I'd document the reasoning behind it here as well.